gnomad_constraint/pipeline/constraint_pipeline.py

```diff
@@ -247,6 +247,10 @@ def main(args):
         # TODO: Add chromosome X back in after complete evaluation for autosome_par.
         regions.remove("chrx_nonpar")
 
+    MODEL_TYPES = constraint_res.MODEL_TYPES
```

Comment:
I don't love changing globals like this unless it's at the top and not done within a function. Would you be OK with adding a models parameter to get_constraint_resources?